docker-entrypoint.sh

```diff
@@ -14,6 +14,12 @@ if [[ $DOCKER_HOST == unix://* ]]; then
 	fi
 fi
 
+# Compute the DNS resolvers for use in the templates
+export RESOLVERS=$(awk '$1 == "nameserver" {print $2}' ORS=' ' /etc/resolv.conf | sed 's/ *$//g')
```

Comment:
I got this

```sh
export NAMESERVERS=$(cat /etc/resolv.conf | grep "nameserver" | awk '{print $2}' | tr '\n' ' ')
```

from https://trac.nginx.org/nginx/ticket/658#comment:5

Not sure which solution is better...